README.md

```diff
@@ -5,41 +5,87 @@
   </a>
 </p>
 
-# Docker Image
-    
-**1. Run a container with PHPSchool.**
-  
+# Getting Started
 
-```bash
+This process will provide you with a Docker environment where you can quickly run PHP School Workshops. We've gone ahead and pre-installed a few for you such as `learnyouphp`!
 
-  docker run -i -t -v ~/Dropbox/docker-phpschool:/phpschool/code:rw -e "PATH=/root/.php-school/bin:$PATH" rafaelcgstz/phpschool bash
+Once you get it up and running you should complete your tasks inside the `code` directory and use an editor such as [Atom](https://atom.io/) to write them.
 
-```
+Everything in the `code` directory will be mounted inside the Docker container at `/phpschool/code` so you can navigate there using the `cd` command and verify your solutions!
 
-**2. To verify your code.** 
-	
-In the docker container, navigate to `/phpschool` folder.
+## Docker Installation
 
-```bash
-  learnyouphp verify <name-of-solution-file>
+Head on over to [Docker](https://www.docker.com/products/docker) and download the latest version for your OS.
 
-```
-    
+#### Special Windows Instructions...
 
-**3. Advanced**
+Older Windows users may require [Docker Toolbox](https://www.docker.com/products/docker-toolbox) instead.
```

Comment:
Do we know what versions require toolbox ? Like "if you are less than 10 you need this" ?